cumulusci/robotframework/utils.py

```diff
@@ -229,3 +233,34 @@ def wrapper(*args, **kwargs):
             raise
 
     return wrapper
+
+
+def get_locator_module_name(version):
+    """Return module name of locator file for the specified version
+
+    If a file for the specified version can't be found, the locator
+    file with the highest number will be returned. Passing in None
+    effectively means "give me the latest version".
+    """
+
+    here = Path(__file__).parent
+    if Path(here, f"locators_{version}.py").exists():
+        # our work here is done.
+        locator_name = f"locators_{version}"
+    else:
+
+        def by_num(filename):
+            """Pull out the number from a filename, or return zero"""
+            nums = re.findall(r"_(\d+)", str(filename))
+            if nums:
+                return int(nums[-1])
+            return 0
+
+        files = sorted(here.glob("locators_*.py"), key=by_num)
+        locator_name = files[-1].stem
+        robot_api.logger.warn(
```

Comment:
@boakley no big deal, but assuming this follows the normal Python logging interface, `warn` is deprecated; `warning` is preferred: https://docs.python.org/3/library/logging.html#logging.Logger.warning